fastmap.c

```diff
@@ -302,7 +303,12 @@ int main_mem(int argc, char *argv[])
 		fprintf(stderr, "\n");
 		fprintf(stderr, "       -v INT        verbosity level: 1=error, 2=warning, 3=message, 4+=debugging [%d]\n", bwa_verbose);
 		fprintf(stderr, "       -T INT        minimum score to output [%d]\n", opt->T);
-		fprintf(stderr, "       -h INT[,INT]  if there are <INT hits with score >80%% of the max score, output all in XA [%d,%d]\n", opt->max_XA_hits, opt->max_XA_hits_alt);
+		fprintf(stderr, "       -h INT[,INT]  if there are <INT hits with score >%.2f%% of the max score, output all in XA [%d,%d]\n", 
+				opt->XA_drop_ratio * 100.0,
+				opt->max_XA_hits, opt->max_XA_hits_alt);
+		fprintf(stderr, "                     A second value may be given for alternate sequences.\n");
```

Comment:
@lh3 it always confused me why two values are allowed.  Can you make sure this matches with your understanding?